Clarifying param use (#8196)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Get-CsAutoAttendant.md

### -Identity
The identity for the AA to be retrieved. If this parameter is not specified, then all created AAs in the organization are returned.

> If you specify this parameter, you can not specify the other parameters.

Reviewer: ```suggestion
If you specify this parameter, you can't specify the other parameters.
```
